recipes/wgbs_tools/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -eoux pipefail
+
+git clone https://github.com/nloyfer/wgbs_tools.git
```

Comment:
This is not good! You should not git clone the master/main branch here like this.
You should use `source>url` in meta.yaml to point to https://github.com/nloyfer/wgbs_tools/tree/0.2.2 and specify `source>sha`.
Then here in build.sh you could use `cd $SRC_DIR` instead of `cd wgbs_tools`

Reply:
👍 done